.snyk

```diff
@@ -2,4 +2,48 @@ exclude:
   global:
     - "vendor/**"
     - "**/*_test.go"
-
+ignore:
+  'snyk:lic:golang:github.com:hashicorp:vault:api:MPL-2.0':
```

Comment:
Can we do something like `snyk:lic:golang:github.com:*:MPL-2.0` here?

Reply:
Tried, not working